model/common/tests/grid_tests/test_topography.py

```diff
@@ -36,14 +37,11 @@ def test_topography_smoothing_with_serialized_data(
     topography_smoothed_verif_np = topography_savepoint.topo_smt_c().asnumpy()
```

Comment:
can you make the test `backend dependent?
```suggestion
    xp = data_alloc.import_array_ns(backend)
    topography_smoothed_ref= topography_savepoint.topo_smt_c().asnumpy()
```